ubuntu-headless-qt/include/02_prepare_rootfs_qt.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function rootfs_qt use to copy binaries of rootfs_qt to ubuntu os.
```

Comment:
This description looks like a function description. Can you update it and move the old description to the correct location of the mentioned functions?